lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -695,6 +695,15 @@ void LogUnimplementedTypeKind(const char *function, CompilerType type) {
 #endif
 }
 
+CompilerType GetCanonicalClangType(CompilerType type) {
+  for (unsigned i = 0; i < 32; ++i) {
```

Comment:
Why 32? Is this just an arbitrary limit? If so I'd raise it to 1024 or something like that...

Reply:
It's an arbitrary limit to guard against cycles.
It's for chains of typedefs, so practically I wouldn't expect more than two to ever appear, but I'm fine with dialing up the limit a bit.